quick-start/src/main/java/com/marklogic/quickstart/service/DataHubService.java

```diff
@@ -105,7 +105,7 @@ public void reinstallUserModules(HubConfig config, DeployUserModulesListener dep
         long startTime = PerformanceLogger.monitorTimeInsideMethod();
 
         try {
-            dataHub.clearUserModules();
+            //dataHub.clearUserModules();
```

Comment:
@akshaysonvane  @srinathgit  This change from Alex seems to have fixed the problem where, when deploying from QS, nothing would be loaded from src/main/ml-modules.

Reply:
QS tooltip for Redeploy says it will clear out the modules DB and reload them. Also removing this will break synchronization between the filesystem and server if the user removes a module from the filesystem.
The problem arises because we are relying on the custom user-modules-timestamp file for force loading the modules but the entries for this modules are in the module-timestamps file of ml-javaclient-util
A quick fix would be to delete both files while force loading 
or
Should we do away with the custom file and start relying on the module-timestamps file of ml-javaclient-util?

Reply:
Let's keep both files in place for now - I took a brief look at getting rid of the hub-specific one, but I wasn't sure of the impacts. Something to consider post 4.1.0.